homeassistant/components/solax/sensor.py

```diff
@@ -75,6 +90,12 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     async_add_entities(devices)
 
 
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Platform setup."""
+    api = await real_time_api(config[CONF_IP_ADDRESS], config[CONF_PORT])
+    await __async_private_setup(hass, async_add_entities, api)
```

Comment:
Should convert it to config flow setup with
```

        hass.async_create_task(
            hass.config_entries.flow.async_init(
                DOMAIN,
                context={"source": config_entries.SOURCE_IMPORT},
                data=device_config,
            )

```
Then async_step_import will bed called

Reply:
Done